src/qtgui/bookmarks.h

```diff
@@ -84,6 +84,15 @@ struct BookmarkInfo
     {
         return frequency < other.frequency;
     }
+
+    bool operator==(const BookmarkInfo &other) const
```

Comment:
Where is this used? Sort?

Reply:
As you edited. :smile: This is needed for `indexOf`.